config/collections.yml

```diff
@@ -151,16 +217,24 @@ default:
     MergeKey: ''
   - Site: https://github.com
     Org: ansible-collections
-    Repo: community.qradar
-    Regex: ''
-    NewCollection: 'TRUE'
-    MergeKey: ''
+    Repo: community.rabbitmq
   - Site: https://github.com
     Org: ansible-collections
-    Repo: community.rabbitmq
     Regex: rabbitmq
```

Comment:
This entry has no "Repo" key any more.

Reply:
Oops, that looks like a copy/paste mistake, fixed in https://github.com/ansible-community/stats-crawler/pull/1/commits/8f02ee497b65534afc5977d060b0d93fcec479f8 thanks !